platform/mds/mds-web/src/main/resources/webapp/partials/modals/import-entity.html

```diff
@@ -6,17 +6,11 @@
                 <h4 class="modal-title" id="importInstanceModalLabel">{{msg('mds.form.importCsv')}}</h4>
             </div>
             <div class="modal-body">
-                <form id="importInstanceForm" name="importInstanceForm" class="form-horizontal" action="getMdsEntityCsvImportPath(selectedEntity.id)" enctype="multipart/form-data" method="POST">
-                    <div class="form-group fileupload fileupload-new" data-provides="fileupload">
+                <form id="importInstanceForm" name="importInstanceForm" class="form-horizontal" action="{{getMdsEntityCsvImportPath(selectedEntity.id)}}" enctype="multipart/form-data" method="POST">
+                    <div class="form-group fileinput fileinput-new" data-provides="fileinput">
                         <label class="col-sm-3 control-label">{{msg('mds.import.file')}}</label>
                         <div class="form-inline col-sm-9">
-                            <div class="form-control input-auto uneditable-input input-xlarge fileupload-exists"><span class="glyphicon glyphicon-file"></span>&nbsp;<span class="mds-import fileupload-preview"></span></div>
-                            <span class="btn btn-default btn-file">
-                                <span class="fileupload-new">{{msg('mds.import.selectFile')}}</span>
-                                <span class="fileupload-exists">{{msg('mds.import.changeFile')}}</span>
-                                <input type="file" id="csvFile" name="csvFile" class="input-file" accept=".csv"/>
-                            </span>
```

Comment:
I see that we lost the feature - accepting only csv file. Is there any easy way to add this again ?